Calc/Calc.hpp

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <iostream>
+
+using namespace std;
+class Calc{
+	    public:
+               Calc();
+               int addNumbers(int, int);
```

Comment:
As this is interface file  Some comment expected to explain the user what the interface does